src/Apollo_Client/core/Apollo_Client__Core_Types.re

```diff
@@ -0,0 +1,27 @@
+module Graphql = Apollo_Client__Graphql;
+
+module ApolloQueryResult = {
+  module Raw = {
```

Comment:
this also would be called JS type right? maybe we could add some comments explaining a bit more to help future contributors

Reply:
Yes. This would be `JS` or `Js_`. Now that you bring this up, I think it's possible it would be clearer to future contributors if there was a clear 1:1 `Bindings` module rather than mixing this stuff, but who know. I'll try a branch off this PR and see if people like that direction.

Reply:
Re: explaining a bit more to help future contributors, I think a `CONTRIBUTING.md` with some explanation of patterns used might be a good goal. https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors